frontend/src/components/InputText.js

```diff
@@ -84,19 +84,12 @@ class InputText extends Component {
               ref={(a) => (this._inputElement = a)}
               placeholder="New Ingredient"
             ></input>
-            <button type="submit">add</button>
+            <Button id="add-btn" type="submit">Add</Button>
+            <Link to="/recommendations">
+              <Button variant="primary">Confirm</Button>
```

Comment:
Could you make the button disabled if user didn't input anything? It could be something like: `<Button disabled={this.state.items.length === 0} ..`